aspnet-core/src/AbpCompanyName.AbpProjectName.Web.Mvc/Controllers/RolesController.cs

```diff
@@ -22,23 +21,21 @@ public RolesController(IRoleAppService roleAppService)
 
         public async Task<IActionResult> Index()
         {
-            var roles = (await _roleAppService.GetRolesAsync(new GetRolesInput())).Items;
             var permissions = (await _roleAppService.GetAllPermissions()).Items;
             var model = new RoleListViewModel
             {
-                Roles = roles,
```

Comment:
Roles property was no needed anymore because i used js service proxies with Datatables.net to bind roles. [Here](https://github.com/iyilm4z/module-zero-core-template/blob/352a70cea875437b17ea66498df809f2dbbacd3a/aspnet-core/src/AbpCompanyName.AbpProjectName.Web.Mvc/wwwroot/view-resources/Views/Roles/Index.js#L17) you can see it.

Reply:
Since you change lib folder name to libs, there are tons of js,css changes which I can not check. 

What about we revert folder name to lib, than change it's name in another pr after we more that one?

Reply:
Sure dear friend, btw thanks for your informative reviewing. I'll do it with a new change set soon.

Reply:
Done. [Here](https://github.com/aspnetboilerplate/module-zero-core-template/pull/489/commits/9450ce6c92131a040a55eb33f8aa5ea6fc6e0d7e) it is.

Reply:
I thought it would reduce the number of file changes, but there was hardly any number change since all css js files changed.
Sory about that 😄

Reply:
:)

Reply:
I reverted it. [Here](https://github.com/aspnetboilerplate/module-zero-core-template/pull/489/commits/07da58a78aab9888330f743a7194f9c400784c5f) it is.